data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -94,6 +94,7 @@ highway-trunk-bridge::shield                        # shield::shield z10- (also
 highway-trunk-tunnel::shield                        # shield::shield z10- (also has pathtext z10-, line z7-, line(casing) z12-)
 === 6740
 
+boundary-aboriginal_lands                           # caption z10- (also has area z8-)
```

Comment:
Where would you suggest the priority for the caption be placed?

Reply:
I think just below the `place-isolated_dwelling`.

Then in case of both features being too close to each other, `boundary-aboriginal_lands` label will be visible still on z12 (as `place-isolated_dwelling` is z14-), while more detailed features will have a chance to show up on more detailed zooms.

Reply:
I think the names of the Indigenous Lands warrants the same priority as other protected areas though. Particularly as they can encompass a wide area such as [Navajo Nation](https://www.openstreetmap.org/relation/9795457#map=8/35.411/-108.600) which according to [Wikipedia](https://en.wikipedia.org/wiki/Navajo_Nation) is home to more than 165,000 people.

Reply:
Yes, but there is a big difference - other protected areas use icons, so this (high) priority is applied to an icon only. A caption that is bound to an icon is optional, i.e. if there is another (even minor) feature nearby then the caption will be hidden, but icons of both the protected area and a nearby feature will stay.

This behavior provides a good balance between importance of the protected area and avoiding its possibly big/long caption displacing too many other features around.

Reply:
But I notice that just below protected areas there is this priority:
```
landuse-reservoir                                   # caption z10- (also has area z12-)
natural-water-lake                                  # caption z10- (also has area z1-)
natural-water-reservoir                             # caption z10- (also has area z1-)
=== 6650
```
These are captions at z10 as well and they seem fine

Reply:
Usually there are no other POIs over water areas :) so such captions are very unlikely to displace anything.